butterknife-gradle-plugin/src/main/groovy/butterknife/plugin/ButterKnifePlugin.groovy

```diff
@@ -5,6 +5,7 @@ import com.android.build.gradle.LibraryPlugin
 import com.android.build.gradle.TestPlugin
 import com.android.build.gradle.api.BaseVariant
 import com.android.build.gradle.api.BaseVariantOutput
+import groovy.json.StringEscapeUtils
```

Comment:
I would prefer not to use Groovy classes since I'll be abandoning the language for something more sane very soon. Can we do something a bit more manual instead?